app/views/ctc/questions/not_filing/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<% @main_question = t("views.ctc.questions.not_filing.title") %>
```

Comment:
@caitbot Do you think we should adjust the position in the I18N tree such that `t('.title')` works? We'd seemingly have to do that for all CTC questions.